config/config.example.yml

```diff
@@ -382,10 +382,14 @@ vocabularies:
     vocabulary: 'srsc'
     enabled: true
 
-# Default collection/community sorting order at Advanced search, Create/update community and collection when there are not a query.
-comcolSelectionSort:
-  sortField: 'dc.title'
-  sortDirection: 'ASC'
+# Common community & collection page config
+comcol:
```

Comment:
I wanted to point out that the old `comcolSelectionSort` config section was **not used** on the Community & Collection pages.  Instead it was used in the Create/Update of new Communities/Collections.
 
I find it a bit odd to lump that setting together with a setting that is specific to Community/Collection pages.  I'm not against it entirely, as I like simplification. But it just needs to be well explained that these configs relate to *both* Communities & Collection but not necessarily just the Community/Collection pages.

The only alternative I can think of would be to leave this `comcolSelectionSort` setting as-is for now & perhaps consider creating your new configs in two places:
```
# This already exists in config.yml
collection;
    searchSection: 
        showSidebar: true

# This does not yet exist, but arguably could.
community:
    searchSection: 
        showSidebar: true
```

That'd allow for more fine-grained control.  But, I admit, I'm not sure if that's necessary for this feature.  So, I won't object to what you've done, but wanted to point out an alternative.